.gitignore

```diff
@@ -9,7 +9,7 @@
 config/starmap/starmap.json
 
 # Ignore compiled linux libs in the root folder, e.g. librust_g.so
-/*.so
+#/*.so
```

Comment:
this wasn't actually the problem I was having but I'm wondering if I should leave the so file in anyways to be nice to that one guy who's trying to develop on linux